internal/aks/check.go

```diff
@@ -0,0 +1,180 @@
+package aks
```

Comment:
We should not be using `internal` packages. This should be moved to the `pkg` dir

Reply:
Ok, I agree, but it will be done in separate PR